### DISCUSSION TITLE
<jemalloc>: Unsupported system page size, memory allocation of 5 bytes failed

### DISCUSSION THREAD

**Discussion by doki23** [databendlabs/databend#4713] [Q&A] [answered]
I followed by the official direction of 'Deploy Databend With Local Disk' and met this problem when launched the meta service.
> \<jemalloc\>: Unsupported system page size
memory allocation of 5 bytes failed

version: databend-v0.7.4-nightly-aarch64-apple-darwin

my device: Darwin doki-mbp.local 21.1.0 Darwin Kernel Version 21.1.0: Wed Oct 13 17:33:01 PDT 2021; root:xnu-8019.41.5~1/RELEASE_ARM64_T6000 arm64

How to resolve it?

**Comment by sundy-li** [comment]
Reproduced in my M1 macbook. But it works fine in local build.

I believe it may be related to cross build. Need future investigation.

**Reply by doki23** [reply]
Thanks, I am going to try local build.

**Reply by xudong963** [reply]
A related ticket: https://github.com/datafuselabs/databend/pull/4721 @doki23

**Comment by bohutang** [comment, marked as answer]
Fixed in https://github.com/datafuselabs/databend/pull/4721, please try 0.7.10:
https://github.com/datafuselabs/databend/releases/tag/v0.7.10-nightly

**Reply by doki23** [reply]
Thanks! It works.
